Patch/figures (#2027)
grommet/hpe-design-system: aries-site/src/pages/components/namevaluelist.mdx

2. **Value**  
   Value is the description of data or a pointer to the location of that data. The format of the value can vary, depending on the data set. Content can include: single line description, multi line description, anchors, array or list of items.

> <Example caption="Example showing the ways value can include different types of content." pad="small" previewWidth="large" height={{ min: 'small' }}>

Reviewer: ```suggestion
<Example caption="Example showing that value can include different types of content." pad="small" previewWidth="large" height={{ min: 'small' }}>
```
